templates/theme-footer.html

```diff
@@ -3,10 +3,14 @@
   from django.core.urlresolvers import reverse
   from django.utils.translation import ugettext as _
   from edxmako.shortcuts import marketing_link
+  from django.utils.timezone import now
 %>
 <div class="wrapper-footer">
   <footer id="footer-openedx">
     <div class="colophon">
+      <div class="copy-colophon">
+        © ${now().strftime('%Y')} <span class="license-text">${_("Stanford Medicine")}</span>
```

Comment:
For consistency in Python/JS, we use single quotes for non-interpolated strings; `$(_('Stanford Medicine'))`.